### PR TITLE
10.3.2 image updage

### PR DESCRIPTION
**Pull request by voroyam**
## 2020-01-02

* Changed
- Upgrade ownCloud version from 10.3.1 to 10.3.2
- Upgrade LDAP, Collabora, OnlyOffice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ For a guide how to get started with this Docker image please take a look at our
 
 ## Versions
 
+[Docker Hub](https://hub.docker.com/r/owncloud/appliance/tags)
+
 * `latest` available as `owncloud/appliance:latest`
```
Comment (tboerger) [inline README.md]: Please update the version listing

File: README.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ For a guide how to get started with this Docker image please take a look at our
 
 ## Versions
 
+[Docker Hub](https://hub.docker.com/r/owncloud/appliance/tags)
```
Comment (tboerger) [inline README.md]: So far the readme content is consistent across the docker repos. We link nowhere directly to that. Beside that we got the list below.

Comment (voroyam) [reply README.md]: Why not link?

**Comment by voroyam** [comment]
> Thanks @voroyam
> I had the same failing tests on 10.2.1 due to a wrong test framework reference.
> I fixed it here https://github.com/owncloud-docker/server/pull/164/files#diff-74dc60589e0610c07e491b31f8693b2bR667 (all the changes in and below this line)
> 
> Can you adapt the apppliance .drone.star too?

sure, as soon as I figure out how to do this :)

**Comment by voroyam** [comment]
I tried to squash the commits but somehow made even more commits.

Need help.

**Pull request closed without merging**